src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -149,9 +149,12 @@ export class TechRecordSummaryComponent implements OnInit {
   handleFormState(event: any): void {
     this.vehicleTechRecordCalculated = cloneDeep(this.vehicleTechRecordCalculated);
 
+    // console.log(this.vehicleTechRecordCalculated.axles.length);
+    // console.log(event.axles.length);
+
```

Comment:
```suggestion
```

Reply:
Yeah thanks sorry, was in a rush pushing this up as I had a meeting :(